coalib/output/ConsoleInteraction.py

```diff
@@ -471,14 +471,17 @@ def require_setting(setting_name, arr):
                          following.
     """
     needed = join_names(arr[1:])
+    section_name = "Section: " + section.name_str + ". "
 
     # Don't use input, it can't deal with escapes!
-    print(colored(STR_GET_VAL_FOR_SETTING.format(setting_name, arr[0], needed),
+
+    print(colored(section_name + STR_GET_VAL_FOR_SETTING.format(setting_name,
+                                                                arr[0], needed),
```

Comment:
could you please post here how it looks now and how it looked before? thanks :D 2 imgurs of terminal suffice.